src/services/vector_store.py

```diff
@@ -0,0 +1,105 @@
+from langchain.document_loaders import TextLoader
+from langchain.embeddings.openai import OpenAIEmbeddings
+from langchain.text_splitter import CharacterTextSplitter
+from langchain.vectorstores import Chroma
+from dotenv import load_dotenv, find_dotenv
+import openai
+import os
+

```

Comment:
The modules of  langchain, chromadb and tiktoken where missing in the requirements.txt. Please add them

Reply:
missing packages have been added to requirements.txt